services/wfm/src/main/java/org/openkilda/simulator/bolts/CommandBolt.java

```diff
@@ -96,7 +96,7 @@ protected void processCommand(Tuple tuple) throws Exception {
                 sw = ((RemoveFlow) data).getSwitchId();
             } else {
                 logger.error("UNKNOWN data type: {}", data.toString());
-                throw new Exception("Unknown command {}".format(data.getClass().getSimpleName()));
+                throw new Exception(String.format(data.getClass().getSimpleName()));
```

Comment:
It's meaningless, you can write `throw new Exception(data.getClass().getSimpleName());` instead.
And the save on 119 line.

Reply:
Or `String.format("Unknown command {}", data.getClass().getSimpleName())`

Reply:
Fixed.